README.md

```diff
@@ -5,34 +5,48 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
 <!-- # sButtons -->
 
 [![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](./LICENSE.md)
- [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](./CONTRIBUTING.md) [![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](./CONTRIBUTING.md) <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](./CONTRIBUTING.md) [![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](./CONTRIBUTING.md) <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-120-orange.svg?style=flat-square)](./CONTRIBUTORS.md)
-<!-- ALL-CONTRIBUTORS-BADGE:END --> 
+
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
 
 :bulb: Simple buttons you can use easily for your next project.
 
 # Participate in Hacktoberfest!
 
 This repository will help people who don't know how to participate in [Hacktoberfest](https://hacktoberfest.digitalocean.com/) or where to start contributing. We will be adding a Hacktoberfest label to help contributors find the issues they can contribute to. If you have any ideas for new issues or ways to help contributors participate in Hacktoberfest, please add them [here](https://github.com/sbuttons/sbuttons/issues/161).
 
-**DO NOT submit any pull request that does no fix based on an issue or an actual bug/fix, as it will be labeled invalid and will not count for your Hacktoberfest participation.**  
+**DO NOT submit any pull request that does no fix based on an issue or an actual bug/fix, as it will be labeled invalid and will not count for your Hacktoberfest participation.**
 
 **New rules have been added to Hacktoberfest participation regarding which PRs and which repositories will count to your participation. Since our repository has the topic "hacktoberfest" then we have opted in for participation in Hacktoberfest. any PR that gets merged or approved by a Member will count to your participation. No labels need to be added.**
 
 # Contents
 
+- [Participate in Hacktoberfest!](#participate-in-hacktoberfest)
```

Comment:
Remove any changes in the contents menu

Reply:
Got it. I think my editor itself is "formatting all this on save". Will fix this asap !
How am I screwing up such an easy PR😂🤦🏻‍♂️